virusscanner/terraform/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.0"
+    }
+  }
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = var.region
+}
+
+resource "aws_vpc" "virus_scanning_vpc" {
+  cidr_block = "10.0.0.0/16"
+  tags = {
+    Name = "Virus scanning VPC"
+  }
+}
+
+resource "aws_subnet" "virus_scanning_subnet1" {
+  availability_zone = "eu-west-2a"
+  vpc_id            = aws_vpc.virus_scanning_vpc.id
+  cidr_block        = "10.0.1.0/24"
+
+  tags = {
+    Name = "Virus scanning subnet for eu-west-2a"
+  }
```

Comment:
Please add extra tags for consistency with the other terraform folder?

Reply:
Done